NEWS

```diff
@@ -73,6 +73,10 @@ CHANGES WITH 255 in spe:
           already use 'prefixstable' addresses with wireless networks, the
           stable address chosen will be changed by the update.
 
+		* The SendHostname and Hostname options are now available for the
```

Comment:
indentation is off

Reply:
Oops. Can we have

```
[NEWS]
indent_style = space
indent_size = 4
```
 in editorconfig maybe?

Reply:
#29735